acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/data/BeaconChainHead.java

```diff
@@ -13,32 +13,62 @@
 
 package tech.pegasys.artemis.test.acceptance.dsl.data;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.primitives.UnsignedLong;
 import org.apache.tuweni.bytes.Bytes32;
 
 public class BeaconChainHead {
+  @JsonProperty("headSlot")
   public final UnsignedLong headSlot;
+
+  @JsonProperty("headEpoch")
   public final UnsignedLong headEpoch;
+
+  @JsonProperty("headBlockRoot")
   public final Bytes32 headBlockRoot;
 
+  @JsonProperty("finalizedSlot")
   public final UnsignedLong finalizedSlot;
+
+  @JsonProperty("finalizedEpoch")
   public final UnsignedLong finalizedEpoch;
+
+  @JsonProperty("finalizedBlockRoot")
   public final Bytes32 finalizedBlockRoot;
 
+  @JsonProperty("justifiedSlot")
   public final UnsignedLong justifiedSlot;
+
+  @JsonProperty("justifiedEpoch")
   public final UnsignedLong justifiedEpoch;
+
+  @JsonProperty("justifiedBlockRoot")
   public final Bytes32 justifiedBlockRoot;
 
+  @JsonProperty("previousJustifiedSlot")
+  public final UnsignedLong previousJustifiedSlot;
+
+  @JsonProperty("previousJustifiedEpoch")
+  public final UnsignedLong previousJustifiedEpoch;
+
+  @JsonProperty("previousJustifiedBlockRoot")
```

Comment:
wow i wouldnt have thought you'd have to annotate here unless the name changes...

Reply:
yep not needed